x11-libs/libva/libva-2.10.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib-minimal
+
+DESCRIPTION="Video Acceleration (VA) API for Linux"
+HOMEPAGE="https://01.org/linuxmedia/vaapi"
+
+if [[ ${PV} = 2.10.0 ]] ; then
+	AUTOCONFIGURED="true"
+else
+	AUTOCONFIGURED="false"
+fi
+
+if ! ${AUTOCONFIGURED}; then
+	inherit autotools
+fi
+
+if [[ ${PV} = *9999 ]] ; then
+	inherit git-r3
+	EGIT_BRANCH=master
+	EGIT_REPO_URI="https://github.com/intel/libva"
+else
+	if ${AUTOCONFIGURED} ; then
+		SRC_URI="https://github.com/intel/libva/releases/download/${PV}/${P}.tar.bz2"
+        else
+		SRC_URI="https://github.com/intel/libva/archive/${PV}.tar.gz -> ${P}.tar.gz"
+        fi
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="MIT"
+SLOT="0/$(ver_cut 1)"
+IUSE="+drm opengl utils vdpau wayland X"
+
+VIDEO_CARDS="nvidia intel i965 nouveau"
+for x in ${VIDEO_CARDS}; do
+	IUSE+=" video_cards_${x}"
+done
+
+RDEPEND="
+	>=x11-libs/libdrm-2.4.46[${MULTILIB_USEDEP}]
+	opengl? ( >=virtual/opengl-7.0-r1[${MULTILIB_USEDEP}] )
+	wayland? ( >=dev-libs/wayland-1.11[${MULTILIB_USEDEP}] )
+	X? (
+		>=x11-libs/libX11-1.6.2[${MULTILIB_USEDEP}]
+		>=x11-libs/libXext-1.3.2[${MULTILIB_USEDEP}]
+		>=x11-libs/libXfixes-5.0.1[${MULTILIB_USEDEP}]
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+"
+PDEPEND="video_cards_nvidia? ( >=x11-libs/libva-vdpau-driver-0.7.4-r1[${MULTILIB_USEDEP}] )
+	video_cards_nouveau? ( >=x11-libs/libva-vdpau-driver-0.7.4-r3[${MULTILIB_USEDEP}] )
+	vdpau? ( >=x11-libs/libva-vdpau-driver-0.7.4-r1[${MULTILIB_USEDEP}] )
+	video_cards_intel? ( >=x11-libs/libva-intel-driver-2.0.0[${MULTILIB_USEDEP}] )
+	video_cards_i965? ( >=x11-libs/libva-intel-driver-2.0.0[${MULTILIB_USEDEP}] )
+	utils? ( media-video/libva-utils )
+"
+
+REQUIRED_USE="|| ( drm wayland X )
+	opengl? ( X )"
+
+DOCS=( NEWS )
+${AUTOCONFIGURED} || DOCS+=( CONTRIBUTING.md README.md )
```

Comment:
I'm still really wondering if we'd be better off just always re-generating autotools files if upstream aren't consistent?

Reply:
Sure we can do it at expense of some flexibility.
* pre-generated `configure` is tested by upstream (but we can hardly expect any errors here)
* skipping of re-generation will speed-up build (but saving is mostly neglectable) 
* no need for autotools to be installed to build (however they are usually installed anyway)

Up to you @thesamesam. I can unify and always re-generate or leave it as is.

Reply:
Let's try regeneration all the time for now. Like you say, most of the time, these tools are present, and it's not a noticeable slow down mostly.

It'd be better if it was non-autotools, but it is what it is. Is that okay with you?

Reply:
OK, will update the PR.